site/docs/selection/init.md

```diff
@@ -5,7 +5,7 @@ title: Initialzing a Selection
 permalink: /docs/init.html
 ---
 
-To initialize a selection, set the `init` property to a map or array of maps from [projected channels or field names](project.html) to the initial value of the selection. Single selections are initialized with a map to single values, multi selections are initialized with an array of maps to single values, and interval selections are initialized with a single map to arrays of values.
+To initialize a selection, set the `init` property to a map or array of maps from [projected channels or field names](project.html) to the initial value of the selection. Single selections are initialized with a map to single values, multi selections are initialized with a single map or an array of maps to single values, and interval selections are initialized with a single map to arrays of values.
```

Comment:
I don't know if users would understand what is "a map", given it's not a JSON construct.  What we are doing is using JSON object to act as a map.

Reply:
Do you prefer to say "mapping" or "dictionary"? Feel free to change this since we are in different time zones 😴

Reply:
Done